99-general-concepts/03-style-guide/01-style-guide/README.md

```diff
@@ -8,8 +8,112 @@
 
 ## Objetivos de Aprendizaje
 
-- ...
+- Usar una guía de estilos para escribir código más legible.
 
 ***
 
-...
+Una guía de estilo es una lista de reglas para formatear su código. Muchas
+empresas usan una guía de estilo para que toda su base de código parezca escrita
+por el mismo desarrollador, en lugar de que los desarrolladores diseñen el
+código de acuerdo con sus propias preferencias. Esto hace que el código sea más
+fácil de leer para todos.
+
+## La guía de estilos Laboratoria
+
+La guía de estilo JavaScript de Laboratoria se puede encontrar
+[aquí](https://github.com/Laboratoria/js-style-guide). Por favor léelo ahora.
+
+Por favor, intente aplicar todas las reglas de la guía Laboratoria en su código
+a partir de ahora. No se preocupe - no necesita memorizarlos todos ahora - puede
```

Comment:
I think if we change the message in the comment above we can delete the first sentence.

- No se preocupe -> No te preocupes
- no necesita -> no necesitas
- puede -> puedes

Reply:
done